### ISSUE TITLE
ERROR(6001) when updating Nemesis Engine

### ISSUE THREAD

**Issue by muhdanwar** [ShikyoKira/Project-New-Reign---Nemesis-Main#554]
The following message was produced when attempting to update engine:
ERROR(6001): An exception has occured while updating. Report to Nemesis' author immediately
Message: No mapping for the Unicode character exists in the target multi-byte code page.

The following mods were selected prior to performing the update:
Blocking Combat Behavior Improvement for Skyrim
Cancelable large stagger
TUDM Attack Cancel
Combat Gameplay Overhaul
Hellblade
Proper Spell Cast Direction
Retimed Hit Frame
Ultimate Combat
The Ultimate Dodge Mod

**Comment by muhdanwar** [comment]
Small update. Message no longer appears when installed 0.83c instead of 0.84c.
The list of mods remained the same between both attemps.

**Comment by muhdanwar** [comment]
Managed to isolate the cause of the problem. It was a player home mod called Schwarky's Proudspire Manor.

The mod has nif files with a multi-byte character in in their names. Specifically the following:
\meshes\Oaristys\Clutter\Mirror°.nif
\meshes\Oaristys\Treasures\TrFlagonSilver°.nif
\meshes\SilverDishesMeal01°.nif

I removed the degree symbol from the filenames and changed the records in the esp via TES5Edit, which solved the problem.

Thanks.
